tests/utils/dynamicBrcode.py

```diff
@@ -0,0 +1,16 @@
+import random
+from starkinfra import DynamicBrcode
+
+
+example_dynamic_brcode = DynamicBrcode(
+    name="Jamie Lannister",
+    city="Rio de Janeiro",
+    external_id="01",
```

Comment:
coloca um gerador de randômicos aqui pra não falhar toda vez que for rodar teste